index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="ru">
+   <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <title>Кошачья энергия</title>
+   </head>
+   <body>
+      <main class="main">
+         <div class="main-screen">
+            <section class="header container">
```

Comment:
Hm, I just thought: why don't we use a `header` tag here?

Reply:
fiiixeeed